runtime/etc/Makefile.mli-shared

```diff
@@ -21,7 +21,13 @@
 
 include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
-CHPL_MLI_INCS = -I$(CHPL_ZMQ_LOC)/include
+ifdef CHPL_ZMQ_HOME
+CHPL_MLI_INCS = -I$(CHPL_ZMQ_HOME)/include
+CHPL_ZMQ_LIB_HOME = $(CHPL_ZMQ_HOME)/lib
+CHPL_ZMQ_DASH_L = -L$(CHPL_ZMQ_LIB_HOME)
+else
+CHPL_ZMQ_LIB = -lzmq
```

Comment:
I realize this is a work in progress, but looked out of curiosity.  Shouldn't this be set independently of whether or not CHPL_ZMQ_HOME is set?

Reply:
That's a blind copy+paste error.  In Makefile.mli-static, we have `libtool` take either the full path to libzmq or this `-lzmq`.  But libtool isn't used in Makefile.mli-shared.  I'm just going to remove that environment variable from Makefile.mli-shared, thanks for pointing it out!